### ISSUE TITLE
harpoon.exe

### ISSUE THREAD

**Issue by kooijman599** [andreldm/harpoond#11]
Where can I find the harpoon.exe after installation??

**Comment by andreldm** [comment]
For windows you don't need to install, just grab the built executable and move it to wherever you want.

**Comment by kooijman599** [comment]
But where can I find the executable then? Sorry, it will be a silly question, but I don't get it.

**Comment by andreldm** [comment]
You need to compile the project itself, here are the [instructions](https://github.com/andreldm/harpoond?tab=readme-ov-file#windows), I don't provide a ready to download binary and at the moment I don't have any windows machine to build it for you. If you install MSYS2 and copy&paste the commands it should work just fine.

**Comment by kooijman599** [comment]
I did earlier. But I really don't know where to like after compiling the project for the .exe file. I searched my whole machine, but there seems to be no harpoon.exe

**Comment by andreldm** [comment]
Once you compile the executable is in the project directory, you can open it with `explorer .`

![image](https://github.com/user-attachments/assets/f2476aa0-0013-41b4-a6e1-3de12823a6d8)

**Comment by andreldm** [comment]
Ok, so I decided to not only build the executable as well as make RGB and DPI values parametrized via program arguments:

[harpoond.zip](https://github.com/user-attachments/files/18268018/harpoond.zip)

This is the [branch](https://github.com/andreldm/harpoond/tree/parameters) (see its README.md) for the new changes, please let me know if it works as intended before I push them into master.

**Comment by kooijman599** [comment]
Unfortunately, the executable starts, but immediately stops again according to Windows Task Manager.

**Comment by andreldm** [comment]
Is any error printed if executed from the windows console?

**Comment by kooijman599** [comment]
No, it just tells: "interrupted" and then rapidly disappear from the Task Manager.

**Comment by andreldm** [comment]
Then it's a dead end for me, I don't have a windows machine, on a VM the executable is stuck, if I add `setbuf(stdout, NULL)` messages are printed normally, but without the mouse (I threw it away a long time ago) it's next to impossible to troubleshoot.

I'm sorry but I can't really help, if you can manage to make it work please let me know.
